src/js/main.js

```diff
@@ -0,0 +1,81 @@
+export const DiscourseRADParser = () => {
+  const radTabsElements = document.querySelectorAll('.rad-tabs-element');
```

Comment:
You should target classes prefixed with `js-` so its clearly disconnects styling from functionality.

Reply:
https://canonical-web-and-design.github.io/practices/coding/stylesheets.html#javascript-namespacing

Reply:
Added `.js-` for all the classes purely used for js functionality. I was not sure if I should add it for `discourse-rad-element` which is also added by the discourse plugin, but this one is used to encapsualte the `scss`, so I chose not to add the prefix.